docs/examples/ex_extract.rst

```diff
@@ -14,11 +14,145 @@ modeled values.
+the fake model variable with the appropriate dimensions (``'dummy2'``).
+
+.. code:: python
+
+   # Set the model dummy variable units
+   model.meta['dummy2'] = {model.meta.labels.units: 'm/s'}
+
+   # Get the xarray data from the model instrument, with metadata attached
+   model_data = pysatModels.utils.convert.convert_pysat_to_xarray(model)
+
+   # Set the extract input parameters
+   input_args = [jro, model_data, ["gdlonr", "gdlatr", "gdalt"],
+                 ["longitude", "latitude", "altitude"], "time", "time",
+                 ["deg", "deg", "km"]]
+   input_kwargs = {'sel_name': ['dummy2']}
+
+   # Run the extract function
+   added_vars = pysatModels.utils.extract.extract_modelled_observations(
+       *input_args, **input_kwargs)
+
+The output from this function will let you know the variable names that were
+added to the observational data :py:class:`~pysat.Instrument`.  If we plot this
+data, we can visualize how the selection occurred.
+
+
+.. code:: pysat
+
+   import matplotlib as mpl
+   import matplotlib.pyplot as plt
+   import numpy as np
+   
+   # Initialize a figure with two subplots
+   fig = plt.figure()
+   ax_alt = fig.add_subplot(211)
+   ax_loc = fig.add_subplot(212)
+
+   # Create plottable model locations
+   mlon, mlat = np.meshgrid(model['longitude'], model['latitude'])
+   mtime, malt = np.meshgrid(model.index, model['altitude'])
+
+   # Get the paired and unpaired JRO indices
+   igood = np.where(~np.isnan(jro[added_vars[0]]))
+   ibad = np.where(np.isnan(jro[added_vars[0]]))
+
+   # Plot the altitude/time data
+   ax_alt.plot(jro.index[ibad[0]], jro['gdalt'][ibad[1]], 'm*',
+               label='JRO unpaired')
+   ax_alt.plot(mtime, malt, 'k.')
+   ax_alt.plot(jro.index[igood[0]], jro['gdalt'][igood[1]], 'r*',
+               label='JRO Pairs')
+
+   # Plot the the lat/lon data
+   ax_loc.plot(mlon, mlat, 'k.')
+   ax_loc.plot(jro['gdlonr'], jro['gdlatr'], 'r*')
+
+   # Format the figure
+   ax_loc.set_xlim(0, 360)
+   ax_loc.xaxis.set_major_locator(mpl.ticker.MultipleLocator(60))
+   ax_loc.set_xlabel('{:} ({:})'.format(
+       model.meta['longitude', model.meta.labels.name],
+       model.meta['longitude', model.meta.labels.units]))
+   ax_loc.set_ylabel('{:} ({:})'.format(
+       model.meta['latitude', model.meta.labels.name],
+       model.meta['latitude', model.meta.labels.units]))
+
+   ax_alt.lines[1].set_label('Model')
+   ax_alt.legend(loc=2)
+   ax_alt.set_xlabel('UT')
+   ax_alt.set_xlim(stime, stime + dt.timedelta(days=1))
+   ax_alt.xaxis.set_major_formatter(mpl.dates.DateFormatter('%H:%M'))
+   ax_alt.set_ylabel('{:} ({:})'.format(
+       model.meta['altitude', model.meta.labels.name],
+       model.meta['altitude', model.meta.labels.units]))
+
+   fig.suptitle('JRO-Test Model Comparison: {:}'.format(
+       stime.strftime('%d %b %Y')))
+   fig.subplots_adjust(hspace=.3)
+
+   # If not working interactively
+   plt.show()
+
+
+And this should show the figure below.
```

Comment:
I've tried this example using `main`, `develop`, and `load_meta_proc`. For pysatModels I've used this branch, and the older `rs_rc_bug` by accident the first time. Anyway, I don't get any of the red points that are in your figure
![Figure_1](https://user-images.githubusercontent.com/3595641/166728373-6bea103c-b46b-4d36-b764-bf05ba6a4a44.png)
.

Reply:
Well, whatever we did that merged into this broke things.

Reply:
Found and fixed the bug.  Not sure why it didn't show up in your plot, but the longitude range was wrong.